pkgs/os-specific/linux/pscircle/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, meson, pkgconfig, ninja, cairo
+}:
+
+stdenv.mkDerivation rec {
+  name = "pscircle-${version}";
+  version = "1.0.0";
+
+  src = fetchGit {
```

Comment:
If the build does not require any special git information, it is better to use `fetchFromGitLab`, since that will download the src much more efficiently.